test/order_test.rb

```diff
@@ -114,48 +114,130 @@
 end
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
       # TODO: Your test code here!
+      orders = Order.all
+      expect(orders.length).must_equal 100
+      expect(orders).must_be_kind_of Array
+      orders.each do |o|
+        expect(o).must_be_kind_of Order
+        expect(o.id).must_be_kind_of Integer
+        expect(o.customer).must_be_kind_of Customer
+        expect(o.fulfillment_status).must_be_kind_of Symbol
+      end
     end
 
     it "Returns accurate information about the first order" do
       id = 1
       products = {
         "Lobster" => 17.18,
         "Annatto seed" => 58.38,
-        "Camomile" => 83.21
+        "Camomile" => 83.21,
       }
       customer_id = 25
-      fulfillment_status = :complete
+      FULFILLMENT_STATUS = :complete
 
       order = Order.all.first
 
       # Check that all data was loaded as expected
       expect(order.id).must_equal id
       expect(order.products).must_equal products
       expect(order.customer).must_be_kind_of Customer
-      expect(order.customer.id).must_equal customer_id
+      expect(order.customer.id).must_equal id
       expect(order.fulfillment_status).must_equal fulfillment_status
     end
 
+
     it "Returns accurate information about the last order" do
       # TODO: Your test code here!
+      id = 100
+      products = {
+        "Amaranth" => 83.81,
+        "Smoked Trout" => 70.6,
+        "Cheddar" => 5.63,
+      }
+      customer_id = 20
+      fulfillment_status = :pending
+
+      order = Order.all.last
+
+      # Check that all data was loaded as expected
+      expect(order.id).must_equal id
+      expect(order.products).must_equal products
+      expect(order.customer).must_be_kind_of Customer
+      expect(order.customer.id).must_equal id
+      expect(order.fulfillment_status).must_equal fulfillment_status
     end
   end
 
   describe "Order.find" do
     it "Can find the first order from the CSV" do
       # TODO: Your test code here!
+      id = 1
+      products = {
+        "Lobster" => 17.18,
+        "Annatto seed" => 58.38,
+        "Camomile" => 83.21,
+      }
+      customer_id = 25
+      fulfillment_status = :complete
+
+      first = Order.find(1)
+
+      expect(order.id).must_equal id
+      expect(order.products).must_equal products
+      expect(order.customer).must_be_kind_of Customer
+      expect(order.customer.id).must_equal id
+      expect(order.fulfillment_status).must_equal fulfillment_status
```

Comment:
You named your order `first` but asserted on `order`.

```suggestion
      order = Order.find(1)

      expect(order.id).must_equal id
      expect(order.products).must_equal products
      expect(order.customer).must_be_kind_of Customer
      expect(order.customer.id).must_equal id
      expect(order.fulfillment_status).must_equal fulfillment_status
```